contracts/IBridgeValidators.sol

```diff
@@ -5,4 +5,5 @@ interface IBridgeValidators {
     function isValidator(address _validator) public view returns(bool);
     function requiredSignatures() public view returns(uint256);
     function owner() public view returns(address);
+    function validatorList() public view returns (address[]);
```

Comment:
Is it valid only for rewardable validators?

Reply:
I'm not sure if I understood the question correctly but `validatorList` is defined on `BaseBridgeValidators` so it is valid for rewardable validator and generic validator contracts

Reply:
I meant that `validatorList` is not needed for generic validator contract since there is no scenarios where it can be used.